src/QMCDrivers/DMC/WalkerControl.cpp

```diff
@@ -27,6 +27,8 @@
 #include "type_traits/template_types.hpp"
 #include "QMCWaveFunctions/TrialWaveFunction.h"
 
+#include "Utilities/for_testing/NativeInitializerPrint.hpp"
```

Comment:
Either move it out of the test space as a generic feature or avoid using it.

Reply:
?

Reply:
removing, sorry thought I'd pushed this.